mis_builder/models/mis_report.py

```diff
@@ -399,6 +399,13 @@ def _compute_field_names(self):
     report_id = fields.Many2one(
         comodel_name="mis.report", required=True, ondelete="cascade"
     )
+    company_field_id = fields.Many2one(
+        comodel_name="ir.model.fields",
+        ondelete="set null",
+        domain="[('model_id', '=', model_id)]",
```

Comment:
This domain seems correct, but I wonder why it is not on the `date_field` above.